### PR TITLE
Add 2.3.5 changes to CHANGELOG.md

### PR DESCRIPTION
**Pull request by daschuer**
This is a first step towards a 2.3.5 release.
Due to some regressions in 2.3.4, we should release it soon. 

The milestone can be watched here:
https://github.com/mixxxdj/mixxx/milestone/39

This is the link to the pending 2.3 PRs: 
https://github.com/mixxxdj/mixxx/pulls?q=is%3Apr+is%3Aopen+base%3A2.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+* macOS: Request Microphone and line-in access permission.
+  [#11367](https://github.com/mixxxdj/mixxx/pull/11367)
+  [#11365](https://github.com/mixxxdj/mixxx/issues/11365)
+* JACK API: Allow to explicit select buffers of 2048 and 4096 frames/period. They are not
+  supported by the automatic buffer setting of the used PortAudio library.
+  [#11366](https://github.com/mixxxdj/mixxx/pull/11366)
+  [#11341](https://github.com/mixxxdj/mixxx/issues/11341)
+* Pioneer DDJ-400: Make Beat FX section more intuitive
+  [#10912](https://github.com/mixxxdj/mixxx/pull/10912)
+* Playlist export: Fix chained file extensions after changing the playlist type.
```
Comment (ronso0) [inline CHANGELOG.md:25]:
```suggestion
* Playlist export: Adopt new extension after changing the playlist type
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [2.3.5](https://github.com/mixxxdj/mixxx/milestone/39) (unreleased)
+
+* Fix empty waveform overview after loading a track. This fixes a Mixxx 2.3.4 regression
+  since [#11162](https://github.com/mixxxdj/mixxx/pull/11162).
```
Comment (ronso0) [inline CHANGELOG.md:6]:
```suggestion
* Fix empty waveform overview after loading a track (Mixxx 2.3.4 regression)
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
Thank you!

I proposed some small improvements.
Would you mind sticking to the one-line style we used previously? Like
`AutoDJ: fix horrible crash [#1098] [#1024]`
and use the multiple-lines list only if there's a significant amount of PRs for a certain topic/issue?
Trailing dots are also not neessary IMHO

**Comment by ronso0** [comment]
LGTM. What do you think about

> Would you mind sticking to the one-line style we used previously? Like
AutoDJ: fix horrible crash [#1098] [#1024]
and use the multiple-lines list only if there's a significant amount of PRs for a certain topic/issue?
Trailing dots are also not neessary IMHO

**Comment by daschuer** [comment]
A, the rendered version is just one line and scales with the available space. So it should be Ok. 

For the source I like to follow the habit to break around 80 columns to have an easy review.

**Comment by daschuer** [comment]
Here: https://github.com/mixxxdj/mixxx/blob/7fa38b05cd0a4138ee21744c1673af159163d1ec/CHANGELOG.md

Oh I fergot that many users will read it as txt. 
In that case we need a reasonable line length. Can we draft a rule for that?

**Comment by Swiftb0y** [comment]
shouldn't be hard to enforce. We're currently disabling it anyways.

https://github.com/mixxxdj/mixxx/blob/45eb0bbddf02187c8ac591bc646680874bf0e822/.markdownlint-cli2.yaml#L18-L20

**Comment by ronso0** [comment]
Oh the changelog.md, of course. I was referring to the metainfo.xml
Anyway, done. Thanks!

**Comment by Swiftb0y** [comment]
Ah sorry, should've payed closer attention, I assumed you mean the markdown file in the context of being viewed by texteditors. I don't think we have much control over how the contents of the metainfo.xml will be displayed anyways.

**Pull request merged**